src/integration_tests/telemetry_async.cpp

```diff
@@ -18,6 +18,7 @@ static void print_armed(bool armed);
 static void print_quaternion(Telemetry::Quaternion quaternion);
 static void print_euler_angle(Telemetry::EulerAngle euler_angle);
 static void print_angular_velocity_body(Telemetry::AngularVelocityBody angular_velocity_body);
+static void print_fixed_wing_metrics(Telemetry::FixedwingMetrics fixed_wing_metrics);
```

Comment:
Now the only change left is to make it consistent. `FixedwingMetrics` translates to `fixedwing_metrics`. Sorry for being a pain :smile:

Reply:
Haha yes, no problem. I will get used for the next features.

Reply:
I think now it's ok